script/Arcueid/Day 13/13_00_ARC13_5.txt

```diff
@@ -563,93 +611,95 @@ A cold wasteland where everything is dead.
 [sha:6fff90333dc79f996d6fb6cabe292f907bc39fc6]{
 -- Page 438, Offset 32931.
 -- ……物事の“死”が視えるという事は、すべてがあやふやで脆い世界に投げ込まれる事だ。
-...To see the 'death' of things is to be thrown into a world where everything is vague and brittle.
+...To see the 'death' of things is to be thrown into a world where everything is brittle and uncertain.
 }
 [sha:6cb4d574f6fe8e6650fe23534a014318b35e02a8]{
 -- Page 438, Offset 32932.
 -- 地面は無いに等しく、空は今にも落ちてきそうなほど、弱々しい。
-So frail that the ground is like nothingness, and the skies could fall at any moment.
+A world so frail that the ground is almost insubstantial, and the sky could fall at any moment.
 }
 [sha:317ff8c604ad6ebf2f2ceda9b6dce6892209c68b]{
 -- Page 439, Offset 32933.
 -- 「なにを―――何の事を言っている、貴様？」
-"What―――what are you saying? You bastard."
+"You―――you bastard, what are you saying?"
 }
 [sha:046f8d9102ff95c2b7ac7a9be215bfdf78e2098b]{
 -- Page 439, Offset 32934.
 -- ロアの声が、動揺している。
-Roa's voice trembles.
+Roa's voice is shaking.
 }
 [sha:c2bd38107e12ea2dff58534ad9a59ce8bf86df17]{
 -- Page 439, Offset 32935.
 -- ……それはそうだろう。
-...That's only natural.
+...It's to be expected.
 }
 [sha:70a73014a18bd76f15ced42af642d83a54702298]{
 -- Page 439, Offset 32936.
 -- 俺の独唱をあいつは一ミリだって理解できない。
-// 独唱 lmao wtf
-He isn't able to comprehend even a single milimeter of my solo performance.
+He can't even begin to understand my monologue.
```

Comment:
![image](https://user-images.githubusercontent.com/16092757/171948319-285118e8-9f5e-4bae-b0d9-b736ef5e73d0.png)

Reply:
Is :jimface: good or bad

Reply:
it's not your fault. You did a great job with what you were given. But it's bad.